eel_hole/templates/partials/search_results.html

```diff
@@ -1,8 +1,19 @@
 {% for r in resources %}
 <div class="block">
   <div class="level">
-    <h2 class="title is-4 level-left">{{r.name}}</h2>
+    <div class="level-left">
+      <span class="title is-4 level-item">{{r.name}}</h2>
+    </div>
   </div>
+  <div class="level">
+    <div class="level-left">
+      {% for source in r.sources %}
+        <span class="tag level-item">database: {{ source["title"] }}</span>
+      {% endfor %}
+    </div>
+  </div>
+  {% if "pudl_parquet" in r.sources|map(attribute="title")|list %}
```

Comment:
What happens if you don't have the `|list` filter in here? I'd expect `map` to output some kind of sequence that accepted `in` already